charts/portworx/values.yaml

```diff
@@ -13,20 +13,20 @@ etcdEndPoint:                         # The ETCD endpoint. Should be in the form
 clusterName: mycluster                # This is the default. please change it to your cluster name.
 usefileSystemDrive: false             # true/false Instructs PX to use an unmounted Drive even if it has a filesystem.
 usedrivesAndPartitions: false         # Defaults to false. Change to true and PX will use unmounted drives and partitions.
-secretType: none                      # Defaults to None, but can be kvdb/k8s/aws-kms/vault/ibm-kp
+secretType: k8s                      # Defaults to None, but can be kvdb/k8s/aws-kms/vault/ibm-kp
 drives: none                          # NOTE: This is a ";" seperated list of drives. For eg: "/dev/sda;/dev/sdb;/dev/sdc" Defaults to use -A switch.
 dataInterface: none                   # Name of the interface <ethX>
 managementInterface: none             # Name of the interface <ethX>
 envVars: none                         # NOTE: This is a ";" seperated list of environment variables. For eg: MYENV1=myvalue1;MYENV2=myvalue2
 
 stork: true                           # Use Stork https://docs.portworx.com/scheduler/kubernetes/stork.html for hyperconvergence.
-storkVersion: 1.2.0
+storkVersion: 2.0.1
 
 customRegistryURL:
 registrySecret:
 
 lighthouse: true
-lighthouseVersion: 1.5.0
+lighthouseVersion: 2.0.1
```

Comment:
Note that lighthouse on https://install.portworx.com/2.0/ is indeed 2.0.1, but it also uses an additional POD (portworx/lh-config-sync:0.2), which here we don't ask the version for.
Please make sure helm-YAML will use 2nd POD (also please let's have QA validate LH-install w/ this change)